.github/workflows/code_review.yml

```diff
@@ -0,0 +1,22 @@
+name: AI Code Reviewer
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+permissions: write-all
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: AI Code Reviewer
+        uses: lfsevergnini/ai-codereviewer@chore/add-model-version-parameter
```

Comment:
The `uses` field in the `AI Code Reviewer` step has been changed to a different repository and branch. Please make sure this change is intentional and necessary.

Reply:
Needed this version to be able to set the model